src/js/components/Investigation/FindingCard.test.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+
+import React from "react"
+
+import FindingCard from "./FindingCard"
+import brim from "../../brim"
+import loginTo from "../../test/helpers/loginTo"
+import provide from "../../test/helpers/provide"
+
+let store
```

Comment:
Isn't it better to stray away from using a global here so these tests can run in parallel?

Reply:
The tests do run in parallel at the file level, instead of the tests level. This pattern is used throughout the tests and works well enough.